src/main/java/com/wolox/training/exceptions/RestExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package com.wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class RestExceptionHandler {
+
+    @ExceptionHandler(NotFoundException.class)
+    public ResponseEntity notFoundException (Exception e){
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(e.getMessage());
+    }
+
+    @ExceptionHandler(IdMismatchException.class)
+    public ResponseEntity idMismatchException (Exception e){
+        return ResponseEntity.status(HttpStatus.CONFLICT).body(e.getMessage());
```

Comment:
Same here, but with "book id mismatch".

Reply:
Fixed in https://github.com/wolox-training/gf-java/pull/9